code/synchronization/forward-trigger-service.py

```diff
@@ -25,12 +25,13 @@
 import serial
 import asyncio
 import logging
-
 import janus
 import keyboard
+import usb.core
+import usb.util
 
 
-LISTEN = 2023
+LISTEN = 8888
```

Comment:
Why this change?

Reply:
I changed it back to 2023.

Reply:
The rationale is that you may have a jupyter notebook running and we don't want to have any conflicts.